src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -90,43 +122,94 @@ export class LogsPageComponent implements OnInit {
     this.formModel.logsTimeTo = { hour: 23, minute: 59 };
   }
 
-  toggleSelection(severity: string): void {
-    this.formModel.logsSeverity.has(severity)
-      ? this.formModel.logsSeverity.delete(severity)
-      : this.formModel.logsSeverity.add(severity);
-  }
-
   searchForLogs(): void {
-    if (this.formModel.logsSeverity.size === 0) {
-      this.statusMessageService.showErrorToast('Please select at least one severity level');
+    if (this.formModel.logsFilter === ''
+      || (this.formModel.logsFilter === this.SEVERITY && this.formModel.logsSeverity === '')
+      || (this.formModel.logsFilter === this.MIN_SEVERITY && this.formModel.logsMinSeverity === '')
+      || (this.formModel.logsFilter === this.EVENT && this.formModel.logsEvent === '')) {
+      this.statusMessageService.showErrorToast('Please select severity level / event');
```

Comment:
I don't think you should combine severity level and event. Even combining severity and min severity is debatable.

Reply:
Will separate them into 3 different error messages